demisto_sdk/commands/common/hook_validations/utils.py

```diff
@@ -0,0 +1,11 @@
+
+def is_v2_file(current_file):
+    """Check if the file is a v2 file
+    Returns:
+        bool. Whether the file is a v2 file
```

Comment:
if applicable just for integrations and scripts please write that in the docstring, otherwise might be misleading.